src/main/java/dao/ItemDaoImpl.java

```diff
@@ -0,0 +1,26 @@
+package dao;
+
+import db.Storage;
+import java.util.Map;
+import java.util.Set;
+import model.Transaction;
+
+public class ItemDaoImpl implements ItemDao {
+
+    @Override
+    public int getBalance(String item) {
+        return Storage.items.get(item);
+    }
+
+    @Override
+    public void process(Transaction transaction) {
+        int previousQuantity = transaction.getOperation() == Transaction.Operation.BALANCE
+                ? 0 : getBalance(transaction.getItem());
+        Storage.items.put(transaction.getItem(), previousQuantity + transaction.getQuantity());
+    }
```

Comment:
please rename methods according to CRUD principles.
C - create.
R - read.
U - update.
D - delete.

Reply:
done